Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatestjournal/TestJournalRepository.kt

```diff
@@ -0,0 +1,15 @@
+package de.rki.coronawarnapp.coronatestjournal
```

Comment:
package coronatestjournal.. (all relevant files)
everything is only needed in Contactjournal package.
No need to create a new separate package

Reply:
updated